### PR TITLE
small doc fixes [ci skip]

### PR DESCRIPTION
**Pull request by softark**

### CODE REVIEW

File: docs/guide/topics-cache.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ return [
 ];
 ```
 
-If you only use the redis cache, you can also configure the parameters of the connection within the
+If you use redis only as a cache, you can also configure the parameters of the connection within the
```
Comment (cebe) [inline docs/guide/topics-cache.md]: should stay as is. It means the you only use the cache class and not e.g. session or AR.

File: docs/guide/topics-cache.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Using the Cache component
 =========================
 
-To use the `Cache` component, in addition to configuring the connection as described above,
+To use the `Cache` component, in addition to configuring the connection as described in [Installation](installation.md) section,
```
Comment (cebe) [inline docs/guide/topics-cache.md]: "in **the** Installation section".

same below.

File: docs/guide/topics-cache.md (hunk 1 above)
Comment (softark) [inline docs/guide/topics-cache.md]: Well, actually I was trying to clarify the same thing.
I thought that "If you only use the redis cache" could be misunderstood as "only the redis cache, i.e. without using  APC, memcache, file cache, etc.".

But, OK, I will try to rewrite it.

**Comment by cebe** [comment]
awesome, thanks!

**Pull request merged**